source/blog/2018-05-25-the-emberjs-times-issue-48.md

```diff
@@ -37,8 +37,17 @@ Halloechen Emberistas!
 
 ---
 
-## [YOUR SECTION TITLE HERE](#your-url-here)
+## Updates to the `ember-cli-addon-docs`(https://github.com/ember-learn/ember-cli-addon-docs)
```

Comment:
If you add the square brackets to the header title you can turn it into a link like so:

```
## [Updates to the ember-cli-addon-docs](https://github.com/ember-learn/ember-cli-addon-docs)
```

which would turn out like this on the blog:

![link](https://user-images.githubusercontent.com/8811742/40433384-3125d85e-5eac-11e8-9e8d-5facd9f93524.gif)